wgpu/src/backend/wgpu_core.rs

```diff
@@ -1352,9 +1352,8 @@ impl crate::Context for ContextWgpuCore {
         device_data: &Self::DeviceData,
         device_lost_callback: crate::context::DeviceLostCallback,
     ) {
-        let device_lost_closure = DeviceLostClosure::from_rust(device_lost_callback);
         self.0
-            .device_set_device_lost_closure(device_data.id, device_lost_closure);
+            .device_set_device_lost_closure(device_data.id, device_lost_callback);
```

Comment:
question: Should we apply `s/callback/closure` for this `device_lost_callback` binding?

Reply:
I did in one of the next commits.

Reply:
So it is, sorry for the noise. 😅 Just barely caught up to that commit.

Wish GitHub could let me submit single comments _and_ comments being held back in a bundle, so I could answer my own naive questions before I submit them. 💦